mmv1/templates/terraform/examples/network_connectivity_hub_policy_mode.tf.tmpl

```diff
@@ -0,0 +1,27 @@
+resource "google_network_connectivity_hub" "{{$.PrimaryResourceId}}" {
```

Comment:
We aim for tests to focus on a single primary resource. You can choose one from these and remove the others. It'd be fine to test one value of the enum field